accepted/PSR-7-http-message.md

```diff
@@ -1637,7 +1640,8 @@ interface UriInterface
      *
      * @param string $host The hostname to use with the new instance.
      * @return self A new instance with the specified host.
-     * @throws \InvalidArgumentException for invalid hostnames.
+     * @throws \InvalidArgumentException for transformations that would result
```

Comment:
You can not change the wording of the docblock in an accepted PSR this is against the bylaws. So the text for the InvalidArgumentException must be reverted.